sandbox/cgroup/tracking_test.go

```diff
@@ -282,13 +282,13 @@ func (s *trackingSuite) TestCreateTransientScopeForTrackingUnhappyNoDBus(c *C) {
 
 	// Calling StartTransientUnit is not attempted without a DBus connection.
 	restore = cgroup.MockDoCreateTransientScope(func(conn *dbus.Conn, unitName string, pid int) error {
-		panic("we are not expecting this call")
```

Comment:
I think this should still do a c.Error() because there are some cases in this code specifically (and others in general) where non-nil errors are ignored and the code does something else because it can try to handle the error in which case the fact that this should not have been called is lost.

Reply:
Ah, interesting observation. I was not aware of `c.Error`.

Reply:
Done